cypress/integration/current_tours_spec.js

```diff
@@ -0,0 +1,57 @@
+describe("Current Tours", () => {
+  const baseUrl = "http://localhost:3000"
+  const toursUrl = "http://localhost:3000/current-tours"
+
+  const logOut = () => {
```

Comment:
This is a good workaround and having it pulled out into this function makes it much more readable!